doc/source/reference/routines.array-manipulation.rst

```diff
@@ -69,10 +69,10 @@ Joining arrays
 
    concatenate
    stack
-   column_stack
-   dstack
-   hstack
    vstack
+   hstack
+   dstack
+   column_stack
    block
```

Comment:
I'd be tempted to put `block` immediately below `stack`.

Reply:
Done.